### DISCUSSION TITLE
Building from source?

### DISCUSSION THREAD

**Discussion by fpolli** [payloadcms/payload#5345] [Q&A] [answered]
Hi, I would like to start experimenting with Payload with the 3.0 alpha. Are there instructions somewhere for building from the source? I cloned the repo, but I don't know what to do next. The instructions I can find are all for 2.0 which is of course released and the instructions all use package downloaders. There is no "build from source" section.

Thank you

**Comment by DanRibbens** [comment, marked as answer]
To experiment with v3 as a dependency in your project, you can look at the repo: https://github.com/payloadcms/payload-3.0-alpha-demo

To run v3 from this monorepo, you can run `pnpm install` and then use `pnpm dev admin`, replace "admin" with any other test suite in the `/test/` directory to run another config, for example `pnpm run fields` has a different config to test.

Does that help?
